cdap-hbase-compat-base/src/main/java/co/cask/cdap/replication/ReplicationConstants.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.replication;
+
+/**
+ * Constants for Replication
+ */
+public class ReplicationConstants {
+
+  /**
+   * Constants for Replication Status Tool
+   */
+  public static final class ReplicationStatusTool {
```

Comment:
I am calling the outer class ReplicationConstants and only the ones specific to the Tools are in this specific class. Leaving room for any other Replication specific constants not related to the tool.